pkg/scaling/scale_handler.go

```diff
@@ -398,6 +398,8 @@ func buildScaler(name, namespace, triggerType string, resolvedEnv, triggerMetada
 		return scalers.NewKafkaScaler(resolvedEnv, triggerMetadata, authParams)
 	case "liiklus":
 		return scalers.NewLiiklusScaler(resolvedEnv, triggerMetadata)
+	case "metrics-api":
```

Comment:
Sorry for missing the discussion on #929. The change looks good to me over all! However given the current implementation, I wonder what you think about `rest-api` instead. I was thinking about `json-api`, but I a future improvement could be adding a `responseType` property that defaults to json, but can handle other encoding or metrics formats.

Reply:
@ahmelsayed using `rest-api` may suggest that users' API must conform to REST design which is not necessary right now. Using `external-api` may be misleading as mentioned here https://github.com/kedacore/keda/pull/1026#discussion_r475606743 so, after all, I think that the current name is most flexible and creates the least confusion

Reply:
fair enough :)